Assets/HotmaxEditorVR/Resources/slowRotation.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections;
```

Comment:
i might move this functionality somewhere more explicit.. I'll loop back around and say where.

Reply:
moved it to the propTile class

Reply:
did you remove the class?